mechanics/SUNSETTING-FEATURES.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Knative feature sunsetting"
+linkTitle: "Sunsetting features"
+weight: 50
+type: "docs"
+---
+
+# Knative Feature Sunsetting
+
+If a feature (especially in alpha stage) is getting no apparent usage and is
+creating mostly busy-work for the community to maintain, the respective working
+group should consider to sunset the respective feature. This needs to be in
```

Comment:
I think there's a larger cost-benefit ratio to consider here. For example, even a beta API which turns out (for example) to have serious scalability implications (KService/cluster from 500 => 80, for example) should be sunsetted.